ttnn/cpp/ttnn/operations/data_movement/concat/device/concat_device_operation.hpp

```diff
@@ -7,13 +7,11 @@
 #include "ttnn/tensor/tensor.hpp"
 #include "ttnn/run_operation.hpp"
 
-namespace tt {
-
-namespace tt_metal {
+namespace ttnn::operations::data_movement {
 
 enum class ConcatOpParallelizationStrategy { MULTI_CORE, SHARDED_MULTI_CORE };
 
-struct Concat {
+struct ConcatDeviceOperation {
```

Comment:
Going forward if you do larger changes like this, follow the latest TMP infra from Akhmed. See Example operation for.. an example

Reply:
I'll do the next PR with the new infra and eventually refactor everything into the new infra